lib/inspec/cli.rb

```diff
@@ -168,8 +168,13 @@ def exec(*targets)
       o[:logger].warn '[DEPRECATED] The use of `--cache` is being deprecated in InSpec 2.0. Please use `--vendor-cache` instead.'
     end
 
-    # run tests
-    run_tests(targets, o)
+    runner = Inspec::Runner.new(o)
+    targets.each { |target| runner.add_target(target) }
+
+    exit runner.run
+  rescue ArgumentError, RuntimeError, Train::UserError => e
```

Comment:
I'm curious what caused you to add this. This will swallow potentially useful stacktraces from dependencies that may throw these types of errors without helpful messaging.

Reply:
This was pulled from the run_tests method so we capture these errors along with the Train specific ones.

Reply:
Got it. As discussed, I think this is fine. What I would love to see in the future is:

 - we capture all exceptions and just hand to `pretty_handle_exception`
 - `pretty_handle_exception` is modified to pint out both the class and the message, and then print the backtrace if log level is debug.

I think this provides a better UX, but totally doesn't have to be done in this PR.